### DISCUSSION TITLE
Reduce boilerplate by using lombok

### DISCUSSION THREAD

**Discussion by mzudse** [remsfal/remsfal-backend#107] [General]
Add lombok to the project, so in new features we can reduce the boilerplate. Already existing boilerplate could then be removed and replaced with lombok in other tickets.

https://projectlombok.org/
https://www.baeldung.com/intro-to-project-lombok

**Comment by astanik** [comment]
Immutables are already used in the core of this project, because no plugin or additional software is needed.

https://immutables.github.io/

Checkout these articles:
https://dev.to/cchacin/immutables-autovalue-lombok-which-one-2j33
https://medium.com/@vgonzalo/dont-use-lombok-672418daa819
